pyznap/send.py

```diff
@@ -64,10 +65,14 @@ def send_snap(snapshot, dest_name, base=None, ssh_dest=None, raw=False):
         send.stderr.close()
 
         stdout, stderr = recv.communicate()
+        if stderr.decode().startswith('cannot receive incremental stream'):
+            raise SSHConnectError(stderr)
```

Comment:
Why not let this be handled by the next line with `CalledProcessError`?

Reply:
I need the SSHConnectError catched by `send_config` in line `337`
when we lost connection, we should do a new `send_filesystem`, if here we raise `CalledProcessError`, `send_snap` and `send_filesystem` will return with 1, `send_config` will think we just get error when send this particular dataset, and will skip it without retry.